pkg/provider/aws/resources/vpc.go

```diff
@@ -88,6 +88,230 @@ type (
 	}
 )
 
+type VpcCreateParams struct {
+	AppName string
+	Refs    []core.AnnotationKey
+}
+
+// Create takes in an all necessary parameters to generate the Vpc name and ensure that the Vpc is correlated to the constructs which required its creation.
+func (vpc *Vpc) Create(dag *core.ResourceGraph, params VpcCreateParams) error {
```

Comment:
Where does import fit in this new structure? Or are we regressing on it for now.

Reply:
regressed for now, but it should fit right in where we check if one is existing we can also ensure if theres an import we have that mapped